resources/lang/de/settings.php

```diff
@@ -48,28 +48,28 @@
     'locale_it' => 'Italienisch',
     'locale_de' => 'Deutsch',
 
-    'security_title' => 'Security',
-    'security_help' => 'Change security matters for your account.',
-    'password_change' => 'Password change',
-    'password_current' => 'Current password',
-    'password_current_placeholder' => 'Enter your current password',
-    'password_new1' => 'New password',
-    'password_new1_placeholder' => 'Enter a new password',
-    'password_new2' => 'Confirmation',
-    'password_new2_placeholder' => 'Retype the new password',
-    'password_btn' => 'Change password',
-    '2fa_title' => 'Two Factor Authentication',
-    '2fa_enable_title' => 'Enable Two Factor Authentication',
-    '2fa_enable_description' => 'Enable Two Factor Authentication to increase security with your account.',
-    '2fa_enable_otp' => 'Open up your two factor authentication mobile app and scan the following QR barcode:',
-    '2fa_enable_otp_help' => 'If your two factor authentication mobile app does not support QR barcodes, enter in the following code:',
-    '2fa_enable_otp_validate' => 'Please validate the new device you\'ve just set:',
-    '2fa_enable_success' => 'Two Factor Authentication activated',
-    '2fa_enable_error' => 'Error when trying to activate Two Factor Authentication',
-    '2fa_disable_title' => 'Disable Two Factor Authentication',
-    '2fa_disable_description' => 'Disable Two Factor Authentication for your account. Be careful, your account will not be secured anymore !',
-    '2fa_disable_success' => 'Two Factor Authentication disabled',
-    '2fa_disable_error' => 'Error when trying to disable Two Factor Authentication',
+    'security_title' => 'Sicherheit',
+    'security_help' => 'Ändere die Sicherheitseinstellungen für dein Konto.',
+    'password_change' => 'Passwort ändern',
+    'password_current' => 'Aktuelles Passwort',
+    'password_current_placeholder' => 'Aktuelles Passwort',
+    'password_new1' => 'Neues Passwort',
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Neues Passwort" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)